docs/developer-docs/latest/getting-started/quick-start.md

```diff
@@ -1,150 +1,488 @@
 
-- Click on **+ Add New Categories** button. Type `French Food` in the **Name** field. Select `Biscotte Restaurant`, on the right in the **Restaurant (0)** dropdown.
-- Click **Save**.
+#### Add a Category to a Restaurant
 
-Navigate back to [**COLLECTION TYPES** - **Categories**](http://localhost:1337/admin/plugins/content-manager/collectionType/application::category.category). You will see the **French Food** category listed in the entries.
+Go to [_Collection types > Restaurants_](http://localhost:1337/admin/plugins/content-manager/collectionType/application::restaurant.restaurant) in the left-hand menu, and click on `Biscotte Restaurant`.
 
-- Click on **+ Add New Categories** button. Type `Brunch` in the **Category** field. **DO NOT ADD** `Biscotte Restaurant` to the `Restaurants` dropdown on the right.
-- Click **Save**.
+In the right sidebar, under **Categories (1)**, click on **Add an item…** and select `Brunch`.  Click **Save**.
 
-You will see the **Brunch** category listed in the entries.
+### Step 3: Set Roles & Permissions
 
-Navigate back to [**COLLECTION TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/collectionType/application::restaurant.restaurant).
+1. Click on _General > Settings_ at the bottom of the left-hand menu.
+2. Under _Users & Permissions Plugin_, choose [**Roles**](http://localhost:1337/admin/settings/users-permissions/roles).
+3. Click the **Public** role.
+4. Scroll down under **Permissions**.
+5. In the **Application** tab, find **Restaurant**. 
+6. Click the checkboxes next to **find** and **findone**.
+7. Repeat with **Category**: click the checkboxes next to **find** and **findone**.
+8. Finally, click **Save**.
 
-- Click on `Biscotte Restaurant`.
-- On the right, under **Categories(1)**, `select` the `Add an item...`, and add **Brunch** as a category for this restaurant, and click the **Save** button.
+![Screenshot: Public Role in Users & Permissions plugin](../assets/quick-start-guide/qsg-handson-part2-04-roles.png)
 
-You have now seen **two different ways** to use the **relation** field type to add and connect relations between Content Types.
+### Step 4: Publish the content
 
-## 7. Set Roles and Permissions
+By default, any content you create is saved as a draft. Let's publish our entries in the collection types.
 
-Navigate to [**GENERAL** > **Settings** > **USERS & PERMISSIONS PLUGIN** > **Roles**](http://localhost:1337/admin/settings/users-permissions/roles).
+First, navigate to [_Collection types > Categories_](http://localhost:1337/admin/plugins/content-manager/collectionType/application::category.category). From there:
 
-- Click the **Public** Role.
-- Scroll down under **Permissions**, open the **Application** tab and find **Restaurant**. Click the checkbox next to **find** and **findone**.
-- Repeat and find **Category**. Click the checkbox next to **find** and **findone**.
-- Click **Save**.
+1. Click the **Draft** button on the `Brunch` category.
```

Comment:
Why the `Draft` button in particular? I think just mentioning the line could be enough.
I'm scared that people think that we mention it, so to be able to update to publish you should have clicked on draft and not on the line. Even if it's going in the same page (the Edit view)

Reply:
This is the feeling I had (that you need to click on the Draft button).
And this also what's written in the current Quick Start Guide.

But I can update it if you want, thanks!